### PR TITLE
scrollable-keyboard-acc: add accessibility support note

### PR DESCRIPTION
**Pull request by WilcoFiers**
This feedback was provided by the ACT Task Force. Firefox it turns out, will add elements with a scrollbar to the sequential focus navigation.

Need for Final Call: none

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ This rule assumes that all [scrollable elements][scrollable] with visible conten
 
 ## Accessibility Support
 
-_No accessibility support issues known._
+Some browsers will automatically make any scrollable element focusable to ensure keyboard accessibility. Those elements become inaccessible only if they have negative number as a tabindex [attribute value][].
```
Comment (maryjom) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md:48]: Suggest editing this to read: 

However, the browser does not automatically make the scrollable element focusable when it has a negative number as a tabindex [attribute value][].

**Review by maryjom — CHANGES_REQUESTED** [review]
See my comment for suggested changes.

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Agree with @maryjom suggestion. Will approve after the change.

**Review by carlosapaduarte — APPROVED** [review]

File: _rules/scrollable-element-keyboard-accessible-0ssw9k.md [hunk 2]
```diff
@@ -45,7 +45,8 @@ This rule assumes that all [scrollable elements][scrollable] with visible conten
 
 ## Accessibility Support
 
-_No accessibility support issues known._
+Some browsers will automatically make any scrollable element focusable to ensure keyboard accessibility. However, the browser does not included these elements in [sequential focus navigation][] when it has a negative number as a tabindex [attribute value][].
```
Comment (jeeyyy) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md:48]: worth linking scrollable element to the glossary item

Comment (jeeyyy) [inline _rules/scrollable-element-keyboard-accessible-0ssw9k.md:48]:
```suggestion
Some browsers will automatically make any scrollable element focusable to ensure keyboard accessibility. However, the browser does not include these elements in [sequential focus navigation][] when it has a negative number as a tabindex [attribute value][].
```

**Review by jeeyyy — DISMISSED** [review]

**Review by Jym77 — APPROVED** [review]

**Pull request merged**